aggregator/Cargo.toml

```diff
@@ -52,7 +50,7 @@ deadpool-postgres = "0.12.1"
 derivative.workspace = true
 fixed = { version = "1.24", optional = true }
 futures = "0.3.30"
-git-version = { version = "0.3.9", optional = true }
+git-version = "0.3.9"
```

Comment:
Since we're only using a macro out of `git-version`, can it be a dev dependency?

Reply:
Yeah, no dice. I think the idea may be that `macro_rules!` must ride along in `.rlib` files, and thus we need Cargo to always build it and pass it along as a dependency to rustc.